tools/test/test_depsolve.py

```diff
@@ -40,6 +40,14 @@ def assert_dnf():
         raise RuntimeError("Cannot import dnf")
 
 
+def is_license_expression_available():
+    if sp.run(
+            ["/usr/bin/python3", "-c", "from license_expression import get_spdx_licensing as _"],
+            check=False).returncode != 0:
```

Comment:
nitpick: The wrapping / line-breaking here confused me for a second.  Maybe cleaner and easier to read if we pull out the cmd part?
```python
    cmd = ["/usr/bin/python3", "-c", "from license_expression import get_spdx_licensing as _"]
    if sp.run(cmd, check=False).returncode != 0:
```

Reply:
I'll add something here: let's not directly refer to `/usr/bin/python3` but replace it with `sys.executable` which points to the Python interpreter that's executing the current code :)

Reply:
> I'll add something here: let's not directly refer to `/usr/bin/python3` but replace it with `sys.executable` which points to the Python interpreter that's executing the current code :)

We should explicitly not do that because we are executing the `osbuild-depsolve-dnf` script using `subprocess`, and it uses `/usr/bin/python3`.

Reply:
Then lets not :)

Reply:
I've added a comment explaining the reason to use `/usr/bin/python3` and used @achilleas-k suggestion WRT formatting.